core/nginx/letsencrypt.py

```diff
@@ -39,8 +45,30 @@
 # Wait for nginx to start
 time.sleep(5)
 
+class MyRequestHandler(SimpleHTTPRequestHandler):
+    def do_GET(self):
+        self.path = '/var/empty/'
+        return SimpleHTTPRequestHandler.do_GET(self)
+
+def serve_one_request():
+    with HTTPServer(("0.0.0.0", 8008), MyRequestHandler) as server:
```

Comment:
Looks like port should be 8080, shouldn't it?

Reply:
No, that was done on purpose to mirror what certbot's standalone uses